### ISSUE TITLE
is it fair to compare the results using or not using instaboost?

### ISSUE THREAD

**Issue by zimenglan-sysu-512** [GothicAi/Instaboost#9]
from the model zoo, the model trained using instaboost is using 4x lr schedule. but the compared one is using 1x. it is fair to compare them?

**Comment by Fang-Haoshu** [comment]
Hi, good question and I believe it is a common concern. 
In fact, if we use 4x for the original config, the margin will be even larger since the original model will be overfitted.  You can check our paper for details. 
In practice, the author of mmdetection also observes the overfitting problem. The performance of Mask R-CNN will degrade even if they train for 2x epoch for strong backbone ResNext-101-64. In our experiments, we found that the performance will degrade more if training for more epochs.

**Comment by Fang-Haoshu** [comment]
We will train some 4x models without InstaBoost and add them to the model zoo.

**Comment by tfzhou** [comment]
Hi Haoshu,

For fair comparison, it will be better if you can report the performance using 1x lr schedule.

**Comment by Fang-Haoshu** [comment]
@tfzhou Hi Tianfei, thanks for your kind suggestion. 

In fact, we found that the performance cannot boost a lot at first, and training longer is needed. From our point of view, the data becomes noisier and more difficult (pattern is more complicated), thus will take longer for the network to learn. I have an idea of training fast-instaboost by first training 1x without InstaBoost, and then boost it for another 1x schedule. I guess it can take shorter time to train if the network is first trained using normal data. But I haven't got time and GPU to try this idea. Will back to you if I trained it:D

**Comment by tfzhou** [comment]
Make sense. Maybe we can try to directly train InstaBoost for another 1x schedule based on well-trained models in mmdetection. If the performance improves, it will also proves the effectiveness of InstaBoost to some extent. I will also try this someday. 

One another question is that from your view, whether InstaBoost can work well in very complex scenarios, e.g., many instances with heavy occlusions. I haven't read the paper very carefully, and I will appreciate if you can give your thought on this question.

**Comment by Fang-Haoshu** [comment]
Hi, thanks~
I guess if the occlusion is severe, InstaBoost may not improve a lot because most of the objects cannot move a lot. But it's basically a guess lol.
